kalite/contentload/management/commands/init_content_items.py

```diff
@@ -0,0 +1,146 @@
+import os
+
+from django.core.management import call_command
```

Comment:
Eventually, this management command can be deprecated, because we can just generate the databases directly from contentload.

At the moment it just transforms our current JSON files into database files.

Due to the default use of bulk inserts, this is a relatively speedy operation.